.github/workflows/pickle.yml

```diff
@@ -87,8 +88,13 @@ jobs:
       - name: Check for existence of pickled files
         id: pickle_check
         uses: ./.github/actions/check_for_pickled
+      - name: Check for STC installation
+        id: stc_check
+        uses: ./.github/actions/check_for_stc
+        with:
+          editable: 'False'
```

Comment:
This value needs to depend on "editable_string". Currently it is False even if the installation is editable

Reply:
Fixed : 12dfa29